File: 250414/남녀노소 구분짓기/sex-and-age.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int gender,old;
    cin >> gender;
    cin >> old;

    if ( gender == 0){
        if(old >= 19)
            cout << "MAN";
        else
            cout << "BOY";
    }
    else{
        if(old >= 19)
            cout << "WOMAN";
        else
            cout << "GIRL";
    }
    return 0;
}

File: 250703/남녀노소 구분짓기/sex-and-age.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int sex = sc.nextInt();
        int n = sc.nextInt();

        if(sex==0){
            if(n>=19)
                System.out.print("MAN");
            else
                System.out.print("BOY");
        }
        else{
            if(n>=19)
                System.out.print("WOMAN");
            else
                System.out.print("GIRL");
        }
    }
}